### PR TITLE
Create error page

### PR DESCRIPTION
**Pull request by RyanHolmesDIT**
No template for error page, needs to be dynamic when an error occurs

### CODE REVIEW

File: src/app/controllers/acs.js [hunk 1]
```diff
@@ -89,6 +89,10 @@ function getIndexData (req, res) {
   })
 }
 
+  function renderError (req, res) {
```
Comment (yozzo) [inline src/app/controllers/acs.js]: We need to remove the `renderError` function, as we will render the error page differently.

File: src/app/routes.js [hunk 2]
```diff
@@ -3,12 +3,15 @@ const ssoController = require('./controllers/sso')
...
   buildFilters,
   renderIndex,
+  renderError,
 } = require('./controllers/acs')
 
 module.exports = function (express, app) {
   app.get('/login/', ssoController.authRedirect)
   app.get('/login/callback/', ssoController.callback)
-
+  app.get('/error/',
```
Comment (yozzo) [inline src/app/routes.js]: We need to  remove the error route call, as we will render the error page differently.

File: src/app/views/error.njk [hunk 3]
```diff
@@ -0,0 +1,25 @@
+{% extends "layout.njk" %}
+
+{% block content %}
+  <div class="govuk-width-container">
+  <main class="govuk-main-wrapper govuk-main-wrapper--l" id="main-content" role="main">
+    <div class="govuk-grid-row">
+      <div class="govuk-grid-column-two-thirds">
+
```
Comment (yozzo) [inline src/app/views/error.njk]: remove empty row

File: src/app/views/error.njk [hunk 4]
```diff
@@ -0,0 +1,25 @@
...
+  <main class="govuk-main-wrapper govuk-main-wrapper--l" id="main-content" role="main">
+    <div class="govuk-grid-row">
+      <div class="govuk-grid-column-two-thirds">
+      
+        <h1 class="govuk-heading-l"> {{error.title}} </h1>
+        <p class="govuk-body"> {{error.message}}
+        </p>
+        <p class="govuk-body">{{error.error}}
+        </p>
+        <p class="govuk-body"> {{error.errors}}
```
Comment (yozzo) [inline src/app/views/error.njk]: remove empty spaces between html tags and nunjucks tags

**Comment by RyanHolmesDIT** [comment]
I have made the changes requested

**Pull request merged**